### PR TITLE
fix(uncouple statusDB Sample reads from CGstats)

### PR DESCRIPTION
**Pull request by ChrOertlin**
## Description

This PR re-routes the data flow from cgstats - cg Sample.reads to cg SampleLaneSequencingMetrics to cg Sample.reads

in addition the temporary column Sample.caluclated_read_counts will be dropped.

### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by ChrOertlin** [comment]
> Nice 👍 At what point will we remove the old flow? Also I'm guessing this PR is not getting merged right away?

The calculated counts are validated with the counts we get from cgstats. If we compare side by side, there are no differences between the values. So we can decouple this now.

File: cg/meta/transfer/flowcell.py [hunk 1]
```diff
@@ -110,8 +110,8 @@ def _parse_flow_cell_samples(
                     sequencing_files=cgstats_sample.fastqs,
                     tag_name=SequencingFileTag.FASTQ,
                 )
-
-            status_db_sample.reads = cgstats_sample.reads
+            # commenting out for now - just for traceability. This code will be removed later.
```
Comment (seallard) [inline cg/meta/transfer/flowcell.py:113]: Oh no zombie code! 🧟

Comment (seallard) [reply cg/meta/transfer/flowcell.py:113]: I guess this entire module is supposed to be nuked so I'm not sure if we should touch it

**Review by seallard — APPROVED** [review]
👍 Maybe we don't need to touch the transfer api at all

File: cg/meta/transfer/flowcell.py (hunk 1 above)
Comment (ChrOertlin) [reply cg/meta/transfer/flowcell.py:113]: yes you are right we have the PRs that keep track of code, I should remove it.

**Comment by ChrOertlin** [comment]
> 👍 I guess we should consider whether there are any cases where the new flow is not run in association with the old flow since that would mean the reads would not be updated for the sample. I can't think of any now.
> 
> We have the force flag implemented and no crashes are occurring for the new flow when run after the old. What about the current manual practice when a sample is assigned the wrong index? Could there be some issue with logic that is only run if the flow cell does not already exist in the new flow?
> 
> Sorry for the vague questions, trying to catch any potential issues with this
> 
> EDIT: After going through the flow again I don't think there should be any issues

We update the counts outside of the flow cell logic, so it should happen any time this command is run. 
Ideally we do not require manual intervention when cgstats is deprecated if I understood the reason correctly why we currently have manual intervention going on.

**Comment by ChrOertlin** [comment]
# Calculcated_read_counts column is gone
image

# Sample gets assigned reads in the reads column when running cg demultiplex finish temporary cmd
image

# cg demultiplex finish temporary runs without issues

**Comment by ChrOertlin** [comment]
stage:
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/proj/stage/demultiplexed-runs
done.
Log deploy... done.
cg, version 40.8.2

prod:

**Pull request merged**